presto-spi/src/test/java/com/facebook/presto/spi/block/TestColumnarMap.java

```diff
@@ -150,11 +152,26 @@ public static BlockBuilder createBlockBuilderWithValues(Slice[][][] expectedValu
         return blockBuilder;
     }
 
-    private static BlockBuilder createMapBuilder(BlockBuilderStatus blockBuilderStatus, int expectedEntries, int expectedBytesPerEntry)
+    private static BlockBuilder createMapBuilder(BlockBuilderStatus blockBuilderStatus, int expectedEntries)
+    {
+        try {
+            return new MapBlockBuilder(
+                    true,
+                    VARCHAR,
+                    VARCHAR,
+                    MethodHandles.lookup().unreflect(Slice.class.getMethod("equals", Object.class)).asType(MethodType.methodType(boolean.class, Slice.class, Slice.class)),
+                    MethodHandles.lookup().unreflect(Slice.class.getMethod("hashCode")).asType(MethodType.methodType(long.class, Slice.class)),
+                    MethodHandles.lookup().unreflect(TestColumnarMap.class.getMethod("blockVarcharHashCode", Block.class, int.class)),
```

Comment:
Use `MethodHandleUtil.methodHandle` to replace `MethodHandles.lookup().unreflect(...)`